scripts/partition_manager/partition_manager.rst

```diff
@@ -812,6 +812,32 @@ You can add or remove partitions as described in the following sections.
 .. note::
   If the static configuration contains an entry for the ``app`` partition, this entry is ignored.
 
+.. _ug_pm_config_dynamic:
+
+Configuring dynamic partitions
+==============================
+
+For child images that use dynamic partition sizing, the Partition Manager allows you to use the :kconfig:option:`CONFIG_PM_PARTITION_SIZE_<CHILD_IMAGE>` Kconfig option to adjust the partition size.
+This is particularly useful when you want to optimize the memory usage of MCUboot to less than 16 kB by enabling cryptographic functionalities.
+
+.. note::
+   The ``CONFIG_PM_PARTITION_SIZE_MCUBOOT`` option should be set in the child image's configuration to match the reduced size.
+
+The :kconfig:option:`CONFIG_PM_PARTITION_SIZE_<CHILD_IMAGE>` Kconfig option allows you to specify the memory size for each child image's partition directly in the Kconfig file.
+This allows for precise control over memory allocation which is crucial for system performance optimization.
+
+Common variants of this Kconfig's option include the following:
+
+* ``CONFIG_PM_PARTITION_SIZE_MCUBOOT``: Sets the partition size for the MCUboot image.
+* :kconfig:option:`CONFIG_PM_PARTITION_SIZE_PROVISION`: Allocates space for the provision partition used in secure boot scenarios.
+* :kconfig:option:`CONFIG_PM_PARTITION_SIZE_SETTINGS`: Defines the size of the settings storage partition, typically used for persistent configuration data.
+* :kconfig:option:`CONFIG_PM_PARTITION_SIZE_SETTINGS_STORAGE`: Sets the size of the settings storage partition.
+* :kconfig:option:`CONFIG_PM_PARTITION_SIZE_NVS_STORAGE`: Configures the size of the non-volatile storage partition.
+
+Each variant must be adjusted according to the specific requirements and memory constraints of the project, ensuring optimal usage of available flash memory.
+For a full list of the :kconfig:option:`CONFIG_PM_PARTITION_SIZE`, see the related `Kconfig search results <https://docs.nordicsemi.com/bundle/ncs-latest/page/kconfig/index.html#!CONFIG_PM_PARTITION_SIZE>`_.
```

Comment:
Is it on purpose that this specifically links to new docs? 
Also the link does not work for me. As you can see it does not show the search you try to link to.
I have tried to link to "unfinished" configurations before with no luck, so I would be interested to see if this is possible
![image](https://github.com/nrfconnect/sdk-nrf/assets/89006246/400be062-dc1a-4bd5-a193-e60a104bffda)

Reply:
I'll look into this - which browser were you using?

Reply:
Firefox.
I can confirm that the link works on chrome

Reply:
*cough* *cough* Also in case of this link, we need to add it to links.txt. *cough*

Reply:
Okay, will do